droid-configs.inc

```diff
@@ -177,6 +177,16 @@ Configure sailfish eg naturally landscape devices like mako
 
 ################################################################
 
+%package flashing
+Provides: on-device-flash-tools
```

Comment:
Name proposals are welcome. Should be probably starting with droid- in the name?